src/server/utils/replication_utils.js

```diff
@@ -61,16 +46,18 @@ function get_copy_type() {
     return 'MIX';
 }
 
-async function copy_objects(scanner_semaphore, client, copy_type, src_bucket_name, dst_bucket_name, keys) {
+async function copy_objects(scanner_semaphore, client, copy_type, src_bucket_name, dst_bucket_name, objects) {
```

Comment:
This will break:
https://github.com/noobaa/noobaa-core/blob/0db21bbf2e1b58a5b62f25525ef95622a020ee6d/src/server/bg_services/log_replication_scanner.js#L209-L220

We need to fix this.